.drone.yml

```diff
@@ -118,23 +158,153 @@ type: kubernetes
 name: deploy-devcenter-prod
 
 steps:
-- name: deploy-prod
-  image: quay.io/mongodb/drone-helm:v3
-  settings:
-    chart: mongodb/web-app
-    chart_version: 4.7.3
-    add_repos: [mongodb=https://10gen.github.io/helm-charts]
-    namespace: devrel
-    release: devcenter-frontend
-    values: image.tag=git-${DRONE_COMMIT_SHA:0:7},image.repository=795250896452.dkr.ecr.us-east-1.amazonaws.com/devrel/${DRONE_REPO_NAME},ingress.enabled=true,ingress.hosts[0]=devcenter-frontend.devrel.prod.corp.mongodb.com
-    api_server: https://api.prod.corp.mongodb.com
-    kubernetes_token:
-      from_secret: prod_kubernetes_token
-    values_files: [ "environments/prod.yaml" ]
+  - name: deploy-prod
+    image: quay.io/mongodb/drone-helm:v3
+    settings:
+      chart: mongodb/web-app
+      chart_version: 4.7.3
+      add_repos: [mongodb=https://10gen.github.io/helm-charts]
+      namespace: devrel
+      release: devcenter-frontend
+      values: image.tag=git-${DRONE_COMMIT_SHA:0:7},image.repository=795250896452.dkr.ecr.us-east-1.amazonaws.com/devrel/${DRONE_REPO_NAME},ingress.enabled=true,ingress.hosts[0]=devcenter-frontend.devrel.prod.corp.mongodb.com
+      api_server: https://api.prod.corp.mongodb.com
+      kubernetes_token:
+        from_secret: prod_kubernetes_token
+      values_files: ["environments/prod.yaml"]
+
+  - name: set-release-sha
+    image: drone/cli:1.4.0-alpine
+    commands:
+    # See comment to similar line above regarding usage.
+    - drone secret update --name app_release_production --data ${DRONE_COMMIT_SHA} mongodb/devcenter
+    environment:
+      DRONE_SERVER: ${DRONE_SYSTEM_PROTO}://${DRONE_SYSTEM_HOST}
+      DRONE_TOKEN:
+        from_secret: drone_token
 
 trigger:
   event:
     - promote
     - rollback
   target:
-    - production
\ No newline at end of file
+    - production
+
+---
+depends_on: null
+kind: pipeline
+type: kubernetes
+name: deploy-devcenter-stage-trigger
+
+steps:
+  - name: checkout-to-current-release
+    image: alpine/git
+    environment:
+      APP_RELEASE:
+        from_secret: app_release_staging
+    commands:
+    # "$${APP_RELEASE}" is the latest staging revision that was promoted successfully.
+    - git checkout "$${APP_RELEASE}"
+
+  - name: publish-staging-from-trigger
```

Comment:
This block shows that 1) `app_release_staging` is the SHA used for the following `publish-staging-from-trigger` build. The following `deploy-staging-from-trigger` step will use the build number produced from the prior step.